spectec/src/util/record.ml

```diff
@@ -0,0 +1,49 @@
+
+(* Helper functions for record *)
+module Record = struct include List
```

Comment:
Why this nested module?

Reply:
Good point, that's because of inexperience in using Ocaml module system 😅.